src/time/wasm.rs

```diff
@@ -0,0 +1,28 @@
+#[cfg(target_family = "wasm")]
+pub struct Instant {
+    start: f64,
+}
+
+#[cfg(target_family = "wasm")]
+impl Instant {
+    pub fn now() -> Self {
+        let window = web_sys::window().unwrap();
+        let performance = window.performance().unwrap();
+        let start: f64 = performance.now();
+        Self { start }
+    }
+
+    pub fn elapsed(&self) -> core::time::Duration {
+        let window = web_sys::window().unwrap();
+        let performance = window.performance().unwrap();
+        let start: f64 = self.start;
+        let now = performance.now();
```

Comment:
performance returns what units ?

Reply:
Performance is this API call from Javascript https://developer.mozilla.org/en-US/docs/Web/API/Performance/now

This is the function https://rustwasm.github.io/wasm-bindgen/api/web_sys/struct.Performance.html#method.now

It requires the Performance feature to be enabled in web_sys. I forgot to include it here.

It returns an f64 representing how many milliseconds since the start of the Javascript runtime (when a worker starts, or when the window starts). In browsers like Brave, it will have millisecond precision sadly. In Chrome, it will have up to nanosecond precision. For security reasons, browsers tend to shift the time by a few ms every minute. If not careful, that could cause unexpected bugs.